<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Equalizer</class>
 <widget class="QWidget" name="Equalizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>waveOverUDP</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="parametricEqGroupBox">
     <property name="title">
      <string>EQ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="f0Label">
        <property name="text">
         <string>f0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="gainLabel">
        <property name="text">
         <string>Gain</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="typeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LowPass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HighPass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BandPassConstSkirt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BandPassConstPeak</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Notch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AllPass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Peak</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LowShelf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HighShelf</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="qSpinBox">
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="f0SpinBox">
        <property name="minimum">
         <double>10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>24000.000000000000000</double>
        </property>
        <property name="value">
         <double>133.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="qLabel">
        <property name="text">
         <string>Q</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="gainSpinBox">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>-10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
